app/services/next_track.rb

```diff
@@ -44,6 +44,6 @@ def self.perform radio
       end
     end
     # liquidsoap's json parser wants strings
-    { cue_out: cue_out.to_i.to_s, cue_in: 0.to_s, fade_out: 0.to_s, fade_in: 0.to_s, track: track.cdn_url }
+    {  cue_in: 0.to_s, fade_out: 0.to_s, fade_in: 0.to_s,cue_out: cue_out.to_i.to_s, track: track.cdn_url }
```

Comment:
Why was this changed?

Reply:
The order of the hashmap is important when checking.

Reply:
Where? It shouldn't matter since you always lookup by key.

Reply:
When rspec checks if those hashmaps are equal to the one hardcoded in, in https://github.com/datafruits/streampusher-api/blob/master/spec/services/next_track_spec.rb,

Reply:
You're right, i have reversed that.